requirements/AUR_CBI_MDL-0130/testing.md

```diff
@@ -0,0 +1,21 @@
+# Testing AUR-CBI-MDL-0130
+
+**1. Steps:**
+
+* Build the project with "make" command
+* Run ./work/binaries/demo file
+
+**2. Expected results:**
+
+* Project should succesfully build
+* The running binary should output logs in a loop with the following lines at the beginning:
+
+    client1: item created 1
+    client1: item created 2
+    client1: item created 3
+    client1: item removed 3
```

Comment:
"[AUR-CBI-MDL-0130]The DataStore component shall have the capability to automatically remove the oldest DataItem when it is full and receiving a new DataItem"
Shouldn't this be item removed 1?

Reply:
That's right, I just copied the actual output. Looks like there is an error in the current implementation. I will change the expected output to what it should be.